### DISCUSSION TITLE
Print listings

### DISCUSSION THREAD

**Discussion by ulankford** [AnthonyBloomer/daftlistings#149] [Q&A] [answered]
Is it possible when searching for residential sales to print more objects?
Or are we limited to title, price and daft_link?

E.g. Number of Bedrooms

**Comment by sdfordham** [comment, marked as answer]
Yes there are many more attributes see the class methods for the [Listing class](https://github.com/AnthonyBloomer/daftlistings/blob/add548f6bfc5ad21b5d1b5797dae482e0d0283af/daftlistings/listing.py#L5).

Here is a MWE for number of bedrooms:
````python
from daftlistings import Daft, Location, SearchType, PropertyType

daft = Daft()
daft.set_location(Location.DUBLIN)
daft.set_search_type(SearchType.RESIDENTIAL_SALE)
daft.set_property_type(PropertyType.HOUSE)
daft.set_min_price(400000)
daft.set_max_price(500000)

listings = daft.search()

for listing in listings:
    print(listing.bedrooms)
````
